src/lsp_data.rs

```diff
@@ -262,6 +262,22 @@ impl Default for InitializationOptions {
     }
 }
 
+// Subset of flags from ls_types::ClientCapabilities that affects this RLS.
+// Passed in the `initialize` request under `capabilities`.
+#[derive(Debug, PartialEq, Deserialize, Serialize, Clone, Copy)]
+#[serde(default)]
+pub struct HandledClientCapabilities {
```

Comment:
Could we call this just `ClientCapabilities` to correspond with the ls_types version?

Reply:
sure. i chose that name to deliberately not confuse it, but if you think there's no chance of confusion, i'll change it.